tests/Unit/Form/Type/AddUserTypeTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Tests\Unit\Form\Type;
+
+use App\Form\AddUserType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\RepeatedType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\Test\TypeTestCase;
+
+class AddUserTypeTest extends TypeTestCase
```

Comment:
Nici in testul asta nu vad valoare, aici tot ce e testat e ca se apeleaza niste metode intr-un punct anume, ceea ce nu mi se pare foarte util, am vrea sa testam rezultatul unei apelari, nu ca apelarea in sine face alte apelari, ca aia se poate schimba si contractul de input/output sa ramana la fel, caz in care nu ne strica noua mare lucru.

Reply:
sterge daca vrei, dar nici nu incurca cu prea multe